modules/v2x/proto/v2x_geometry_def.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto2";
+
+package apollo.obu.normal;
+
+// A point in the map reference frame. The map defines an origin, whose
+// coordinate is (0, 0, 0).
+// Most modules, including localization, perception, and prediction, generate
+// results based on the map reference frame.
+// Currently, the map uses Universal Transverse Mercator (UTM) projection. See
+// the link below for the definition of map origin.
+//   https://en.wikipedia.org/wiki/Universal_Transverse_Mercator_coordinate_system
+// The z field of PointENU can be omitted. If so, it is a 2D location and we do
+// not care its height.
+message PointENU {
+  optional double x = 1 [default = nan];  // East from the origin, in meters.
+  optional double y = 2 [default = nan];  // North from the origin, in meters.
+  optional double z = 3 [default = 0.0];  // Up from the WGS-84 ellipsoid, in
+  // meters.
```

Comment:
Use existing header, geometry and other protos in https://github.com/ApolloAuto/apollo/tree/master/modules/common/proto. Please avoid using duplicate definitions.

Reply:
These proto files are copied from existing files and used as roadside information formats.Since the roadside software update frequency is slower than that of the main line, if the proto file of the main line is updated and the roadside software is not updated, the roadside message parsing will fail.So copy and fix these files and use the _proto adapter_ module to convert to and from the main line message.

这些proto文件复制自已有的文件，用作路侧信息格式。由于路侧软件更新频率比主线慢，若主线proto文件更新，而路侧软件未更新，则会导致路侧消息解析失败。所以采用拷贝并固定这些文件并使用 _proto adapter_ 模块与主线消息进行相互转换。

Reply:
Proto will be designed for backward compatible. So you don't have to worry about that. All modules in Apollo work as a complete system, so re-use existing definitions of proto and gflags is required to minimize the confusion for external developers.